src/data-dictionary/events/Span/networkError.md

```diff
@@ -0,0 +1,8 @@
+---
+name: networkError
+type: attribute
+events:
+  - Span
+---
+
+The error message associated with the iOS NSURL Error code. See networkError.code for more information.
```

Comment:
networkError in code?

Reply:
@urbiz-nr, I added the back ticks for code.